examples/step-49/step-49.cc

```diff
@@ -99,15 +102,41 @@ void print_mesh_info(const Triangulation<dim> &triangulation,
 // discussed in the introduction how to generate it. This follows the same
 // pattern as used in step-5 to load a mesh, although there it was written in
 // a different file format (UCD instead of MSH).
+//
```

Comment:
Here we could use this opportunity to document the possibility to save also manifold ids, when using gmsh api and gmsh > 4.1. If you specify 
```
GMSH_INCLUDE_DIR
GMSH_LIBRARY
```
when building deal.II, then   `DEAL_II_WITH_GMSH_WITH_API` gets defined and you can use 

https://www.dealii.org/current/doxygen/deal.II/group__simplex.html#ga6a87000a17a31132729c23656ada559b

to write and read both manifold and boundary ids by assigning names to the physical ids, as in 

```
$PhysicalNames
4
1 1 "ManifoldID:0"
1 2 "BoundaryID: -1, ManifoldID: 1"
2 3 "ManifoldID: 1"
2 4 "MaterialID: 2, ManifoldID: 1"
$EndPhysicalNames
```